generate_variant_annotation.py

```diff
@@ -247,56 +122,72 @@ def main():
     )
 
     # Sort columns
-    col_order = ['locus_GRCh38', 'chrom_b37', 'pos_b37', 'chrom_b38', 'pos_b38',
-                 'ref', 'alt', 'allele_type', 'vep', 'rsid', 'af', 'cadd']
-    ht = ht.select(*col_order)
-
-    # Persist as writing twice would cause re-computation
-    ht = ht.persist()
+    col_order = [
+        'locus_GRCh38', 'chrom_b38', 'pos_b38',
+        'chrom_b37', 'pos_b37',
+        'ref', 'alt', 'allele_type', 'vep', 'rsid', 'af', 'cadd'
+    ]
 
     # Repartition and write parquet file
     (
-        ht.to_spark(flatten=False)
-          .repartition(out_partitions)
-          .write.parquet(out_parquet)
+        ht
+        .select(*col_order)
+        .to_spark(flatten=False)
+        # .repartition(OUT_PARTITIONS)
+        .coalesce(OUT_PARTITIONS)
+        .write.mode('overwrite').parquet(out_parquet)
+        # .write.format('json').mode('overwrite').option('compression', 'gzip').save(out_parquet.replace('.parquet', '.json.gz'))
     )
 
     # Export site list
-    cols = ['chrom_b37', 'pos_b37', 'chrom_b38', 'pos_b38', 'ref', 'alt', 'rsid']
     (
-        ht.select(*cols)
-          .export(out_sitelist)
+        ht.select(*[
+            'chrom_b37', 'pos_b37',
+            'chrom_b38', 'pos_b38',
+            'ref', 'alt', 'rsid'
+        ])
+        .export(out_sitelist)
     )
```

Comment:
Again just for my understanding, but why do we need to export this separately? This looks like just the same dataset, just with fewer columns

Reply:
It has been raised by @ireneisdoomed and apparently this file is not used anywhere and will be removed from the script.